localstripe/resources.py

```diff
@@ -678,6 +694,28 @@ def _api_delete(cls, id):
         schedule_webhook(Event('customer.deleted', obj))
         return super()._api_delete(id)
 
+    @classmethod
+    def _api_list_sources(cls, id, object=None, limit=10):
+        print(object)
+        try:
+            if object is not None:
+                assert object in ('card', 'bank_account')
+        except AssertionError:
+            raise UserError(400, 'Bad request')
+        
+        obj = cls._api_retrieve(id) # return 404 if does not exist
+        
+        url = '/v1/customers/' + obj.id + '/sources'
+
+        if object is 'card':
+            return Card._api_list_all(url, customer=obj.id)
+
+        #TODO: implement bank accounts
+        if object is 'bank_account':
+            return List(url, limit=limit)
```

Comment:
I propose to simplify all this and don't use the `object` argument. Just check it, but don't use it, and return *all* sources. What do you think?